src/main/java/com/synopsys/integration/detect/workflow/event/Event.java

```diff
@@ -53,4 +53,5 @@
     public static final EventType<UnrecognizedPaths> UnrecognizedPaths = new EventType<>(UnrecognizedPaths.class);
     public static final EventType<SortedMap<String, String>> RawMaskedPropertyValuesCollected = new EventType(SortedMap.class);
     public static final EventType<Operation> DetectOperation = new EventType(Operation.class);
+    public static final EventType<Collection<Operation>> DetectOperationsComplete = new EventType(Collection.class);
```

Comment:
Do we need this new event? Why is the old event not sufficient? If we need this new one, does that mean we don't need the old one?